linode/manager
# M3-5809: Cloud Manager changes for Cypress move to Jenkins

## Description

**What does this PR do?**
Adds a couple package scripts that will simplify the Jenkins work required to run our E2E tests, and cleans up the old workflow file that will no longer be used.

- Creates a new `cy:e2e:wait` package script, which waits for `localhost:3000` to respond before running tests, but otherwise behaves similarly to `cy:e2e`
- Creates a new `cy:ci` package script, which runs `yarn start:manager` and waits for `localhost:3000` to respond before running tests, and kills both processes once the tests complete
- Deletes `e2e_pr.yml` workflow
- Deletes `wait-for-it.sh` script, which served the same purpose as the `npm-run-all` package that this PR adds

## How to test

**What are the steps to reproduce the issue or verify the changes?**
Run `yarn` or `yarn up` to update dependencies (kill `yarn up` afterwards), and then run:

```bash
yarn cy:ci
```

Confirm that:
- The tests run as expected, without the need to manually run `yarn up`
- When the tests are finished, Cypress and webpack-dev-server both quit (check that `localhost:3000` cannot be visited in your browser to confirm that webpack-dev-server is not still running in the background)
